src/ru/geekbrains/HomeWork/Java2/Lesson1/Interface/Treadmill.java

```diff
@@ -0,0 +1,16 @@
+package ru.geekbrains.HomeWork.Java2.Lesson1.Interface;
+
+        import ru.geekbrains.HomeWork.Java2.Lesson1.model.Cat;
+        import ru.geekbrains.HomeWork.Java2.Lesson1.model.Human;
+        import ru.geekbrains.HomeWork.Java2.Lesson1.model.Robot;
+
+public interface Treadmill {
+
+    void RunningHuman(Human human);
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.